toolchains/mi250.hipcc.adastra.spack/toolchain.cmake

```diff
@@ -6,22 +6,6 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 # NOTE: We are not supposed to define CMAKE_BUILD_TYPE here.
 set(CMAKE_BUILD_TYPE Release) # Debug, Release, RelWithDebInfo and MinSizeRel
 
-# Kokkos options
-set(Kokkos_ENABLE_HIP ON CACHE BOOL "Defined if the HIP execution space is enabled.")
-set(Kokkos_ENABLE_HIP_MULTIPLE_KERNEL_INSTANTIATIONS ON CACHE BOOL "If defined, multiple kernel versions are instantiated potentially improving run time.")
```

Comment:
Note that the option `Kokkos_ENABLE_HIP_MULTIPLE_KERNEL_INSTANTIATIONS` has no equivalent in the Spack recipe.